### DISCUSSION TITLE
[Icon Change Request] - Search Filled

### DISCUSSION THREAD

**Discussion by JameelKhan9** [tailwindlabs/heroicons#928] [Ideas]
The search-filled icon does not look any different from the outline icon and causes some accessibility issues.

Can hero icons support a filled variant of the magnifying glass like Spotify, so that when a color is applied it is as noticeable as other completely filled icons?
<img width="148" alt="image" src="https://user-images.githubusercontent.com/59159249/217449315-489023cb-181e-4cd6-bbff-1a737e387e0d.png">

**Comment by zefir-git** [comment]
What accessibility concerns does this cause? I personally would not associate the 2nd icon from the image as "search". It looks more like a radio checkbox or a or table tennis racquet to me.

If you want to make the icon bolder/more visible, you can use the "outline" version and increment the "stroke-width" property. E.g. here I have changed it to `3`, which is double the original.

```svg
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="3" stroke="currentColor" class="w-6 h-6">
  <path stroke-linecap="round" stroke-linejoin="round" d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z" />
</svg>
```

![s](https://user-images.githubusercontent.com/66572326/221111487-3cb37606-8977-4527-8b36-3789dac57f9a.png)
